public/locales/pl/alerts.json

```diff
@@ -3,7 +3,7 @@
   "loginAgain": "Zaloguj się ponownie!",
   "serviceUnavailable": "Usługa niedostępna",
   "unexpectedError": "Nieoczekiwany błąd",
-  "unexpectedErrorMessage": "Your ad blocker may be blocking API calls to last.fm, try disabling it or adding an exception for ws.audioscrobbler.com",
+  "unexpectedErrorMessage": "Twój AdBlocker może blokować API z LAST.FM, spróbuj wyłączyć wtyczkę lub dodaj stronę ws.audioscrobbler.com do wyjątków w AdBlocku.",
```

Comment:
```suggestion
  "unexpectedErrorMessage": "Twój ad blocker może blokować API last.fm, spróbuj wyłączyć wtyczkę lub dodaj wyjątek dla ws.audioscrobbler.com",
```

Reply:
I don't know about the translations, but "ad blocker" is better than "AdBlocker" (as we are not endorsing a particular ad blocker), and last.fm may be all lowercase or "Last.fm", but not all caps. Thanks for bringing this to attention @CvX

Reply:
I'm adding this translation as part of an upcoming commit, thanks to both of you @azertgy @CvX